### PR TITLE
[FIX] failing Detox tests on Xcode upgrade PR

### PR DESCRIPTION
**Pull request by cortisiko**
**Development & PR Process**
1. Follow MetaMask [Mobile Coding Standards](https://docs.google.com/document/d/1VJLwTRsUw_5EDq_o8d6sSbXUAYENLurkRitYO45eY5o/edit?usp=sharing)
2. Add `release-xx` label to identify the PR slated for a upcoming release (will be used in release discussion)
3. Add `needs-dev-review` label when work is completed
4. Add `needs-qa` label when dev review is completed
5. Add `QA Passed` label when QA has signed off

**Description**

This PR fixes the Detox E2E tests for bitrise when we switch devices from the iPhone 11 Pro to the iPhone 12 Pro

**Screenshots/Recordings**

_If applicable, add screenshots and/or recordings to visualize the before and after of your change_

**Issue**

Progresses #???

**Checklist**

* [ ] There is a related GitHub issue
* [ ] Tests are included if applicable
* [ ] Any added code is fully documented

### CODE REVIEW

**Review by NicolasMassart — APPROVED** [review]
I wouldn't have removed all the code comments, but it the tests are still understandable, that's fine for me.
I approve and apologise for the changes required by switching to iPhone 12 pro, I was not expecting it had so large impact.

**Comment by cortisiko** [comment]
Not your fault @NicolasMassart ! The device we use to test was a bit outdated anyways.

File: e2e/specs/permission-system-removing-imported-account.spec.js [hunk 1]
```diff
@@ -87,17 +83,16 @@ describe('Permission System Test: Revoking accounts after connecting to a dapp',
     await ConnectedAccountsModal.tapToSetAsPrimaryAccount();
   });
 
-  it('should submit a dapp transaction ', async () => {
-    await TestHelpers.swipe(BROWSER_SCREEN_ID, 'up', 'slow', 0.2);
-    await TestHelpers.tapAtPoint(BROWSER_SCREEN_ID, { x: 250, y: 420 }); // tapping connect button
+  it('should submit a EIP1559 transaction ', async () => {
+    await TestHelpers.swipe(BROWSER_SCREEN_ID, 'up', 'slow', 0.1);
+    await TestHelpers.tapAtPoint(BROWSER_SCREEN_ID, { x: 320, y: 500 });
```
Comment (sethkfman) [inline e2e/specs/permission-system-removing-imported-account.spec.js:88]: Have you thought about making these values constants at the top of the file to make it easier to update?

Comment (cortisiko) [reply e2e/specs/permission-system-removing-imported-account.spec.js:88]: ahh good call. will make the change now

**Review by sethkfman — APPROVED** [review]
LGTM

**Pull request merged**